src/init/init.ts

```diff
@@ -247,6 +253,41 @@ function createSelectPrompt(env: YeomanEnvironment): InquirerQuestion {
 }
 
 /**
+ * Checks and returns proxy settings from gitconfig. This function
+ * assumes that gitconfig will either be located locally (project's config file)
+ * or globally (user home)
+ */
+function getProxyConfig() {
+  //
+  let pathToConfig = './.git/config';
```

Comment:
hmm, this isn't good to hard-code for everyone. Can you make this the CLI option provided by the user? If no flag & file path given, no proxy used.